### ISSUE TITLE
Make GraphDB chart gitops capable (or: idempotent)

### ISSUE THREAD

**Issue by sjentzsch** [Ontotext-AD/graphdb-helm#118]
The `provisioner` user's password is bcrypted and generated via `htpasswd` (see also: https://github.com/Ontotext-AD/graphdb-helm/blob/main/files/config/users.js#L19). It would get another bcrypt every time Helm renders.

Proposal: Provide a way for the user to optionally, in addition to the password, set the bcrypted-version of the password. If this is set (default can be unset, so we do not change any of the current behavior), then take this, instead of running `htpasswd`.

My workaround for now is to completely write the users.js myself (copied from your version), which is not ideal.

**Comment by mihailradkov** [comment]
I agree with the proposed solution. We'll try to include it in the next minor version.

**Comment by sjentzsch** [comment]
Solved in the meantime, thanks! Closing :-)
